### DISCUSSION TITLE
Test run retention period

### DISCUSSION THREAD

**Discussion by fturibak** [ludeknovy/jtl-reporter#186] [General]
Hello!

Me again :-)

I have a question about Test Retention period

![image](https://user-images.githubusercontent.com/59801436/200786651-4e050694-3cf8-4476-bebd-cec916ac9040.png)

If I mark some test run as a base run will the data for him be deleted after 7/14/30/90 days?

![image](https://user-images.githubusercontent.com/59801436/200787789-e5cf9f29-a151-4567-8328-d7c78d9e43ca.png)

Thing is, that for base run I want to have data persistent ( so that I can all future test runs compare to him )

Thanks in advance

**Comment by ludeknovy** [comment]
Hi @fturibak !

Good question! I checked the query and it will delete the base run as well. I agree that it makes sense to not to delete it.

**Reply by fturibak** [reply]
Could we create an issue to fix this in some future release?

Thanks

**Reply by ludeknovy** [reply]
Sure, this should be easy to fix.

**Reply by ludeknovy** [reply]
https://github.com/ludeknovy/jtl-reporter/issues/187

**Reply by ludeknovy** [reply]
https://github.com/ludeknovy/jtl-reporter/releases/tag/v4.4.7
